tap_s3_csv/csv_iterator.py

```diff
@@ -12,9 +12,23 @@ def get_row_iterator(iterable, options=None):
         (line.replace('\0', '') for line in file_stream), 
         fieldnames=field_names, 
         delimiter=options.get('delimiter', ','),
-        escapechar=options.get('escape_char', '\\'))
+        escapechar=options.get('escape_char', '\\'),
+        quotechar=options.get('quotechar', '"'))
 
     headers = set(reader.fieldnames)
+
+    # Check for duplicate columns
+    if len(reader.fieldnames) != len(headers):
+        fieldname_count = {}
+        duplicate_cols = set()
+        for fieldname in reader.fieldnames:
+            fieldname_count[fieldname] = fieldname_count.get(fieldname, 0) + 1
```

Comment:
why do we care about count? isn't it enough for the if to be `if fieldname in filedname_count:`?

Reply:
fixed.